src/Assignment-4/assignment4_test.js

```diff
@@ -14,7 +14,7 @@
   var quixote = require("../../vendor/quixote.js");
 
   // Define a test suite using Mocha's standard `define` function.
-  describe("Assignment-4", function () {
+  describe.only("Assignment-4", function () {
```

Comment:
Why do we require `.only` here?

Reply:
As per the instruction forwarded in group for Step 2 in "Running test cases"
https://the10x-academy.slack.com/archives/C027J47DQ12/p1637067590217500.

Reply:
That was a temporary workaround to enable the students to be able to run only a particular set of test cases. Let us not have `.only` in the repo. Please remove it.